portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,98 @@
+body {
+  background: #dfe0e1;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+img {
+  max-height: 60vh;
+  width: auto; 
+  border-radius: 15%;
+  opacity: .9;
+  transition-timing-function: ease-in-out;
+  transition: 250ms;
+  padding: 1vh;
+}
+
+.logo img {
+    max-height: 6vh;
+    width: auto;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 67.5vw;
+  padding: 3vh;
+}
+
+
+h1 {
+  display: block;
+  font-size: 10vh;
+  margin-block-start: 0.67em;
+  margin-block-end: 0.67em;
+  margin-inline-start: 10vw;
+  margin-inline-end: 0px;
+  font-weight: bold;
+}
+
+h1 > div {
+  font-size: 8vh; 
+  color: navy;
+}
+
+p {
+  font-size: 5vh;
 }
+#container {
+  text-align: justify;
+  justify-content: space-evenly;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+#container > div {
+  display: inline-block;
+  vertical-align: top;  
+}
+
 
-#greeting-container {
-  margin-top: 20px;
+.centered-over-img {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 6.5vh;
 }
+
+.containerBox {
+  position: relative;
+  display: inline-block;
+  vertical-align: top;
+}
+
+.text-box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  user-select: none;
+}
+
+.text-box:before { 
+  display: inline-block;
+  height: 100%;
+  vertical-align: middle;
+}
+
+container-box-caption {
+  display: inline-block;
+  font-size: 6.5vh;
+  color: white;
+  font-weight: bold;
+}
+
+img:hover { 
```

Comment:
applying uniform styles by HTML tag creates high risk of unintended side effects, you're gonna want to always attach a class or ID to your rules.

In general:  Always limit the scope of your changes by creating some sort of container for what you're doing and saying it only applies in that container.  That way when other people try to do things in the future, they won't have to deal with side effects from your code unless they're working within your container (in which case they'd be aware of that context).

("container" used in the abstract here, not an actual container element).